phylofisher/matrix_constructor.py

```diff
@@ -77,6 +78,7 @@ def run_snakemake(length_filter=False):
         f'--rerun-incomplete',
         f'--keep-going',
         f'--use-conda',
+        '--nolock'
```

Comment:
Just a question, does this need and f in front? a comma after?

Reply:
No it doesn't